src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/repositories/WorkflowRunRepository.kt

```diff
@@ -3,12 +3,19 @@ package org.vaccineimpact.orderlyweb.db.repositories
 import com.google.gson.Gson
 import org.vaccineimpact.orderlyweb.db.JooqContext
 import org.vaccineimpact.orderlyweb.db.Tables
+import org.vaccineimpact.orderlyweb.db.Tables.ORDERLYWEB_WORKFLOW_RUN
+import org.vaccineimpact.orderlyweb.errors.UnknownObjectError
+import org.vaccineimpact.orderlyweb.jsonToGenericList
+import org.vaccineimpact.orderlyweb.jsonToStringMap
+import org.vaccineimpact.orderlyweb.models.WorkflowReportWithParams
 import org.vaccineimpact.orderlyweb.models.WorkflowRun
 import java.sql.Timestamp
 
 interface WorkflowRunRepository
 {
     fun addWorkflowRun(workflowRun: WorkflowRun)
+    @Throws(UnknownObjectError::class)
+    fun getWorkflowDetails(key: String): WorkflowRun
```

Comment:
```suggestion
    fun getWorkflowRunDetails(key: String): WorkflowRun
```
Again for consistency - we already have `addWorkflowRun` and `getWorkflowRunSummaries`.